src/index.js

```diff
@@ -1,31 +1,51 @@
-// import _ from 'lodash';
+import { findCity, findWeather } from './resource';
 
+const searchForm = document.querySelector('form');
+const displayCard = document.querySelector('.display-card');
+const weatherDetails = document.querySelector('.weather-details');
+const time = document.querySelector('img.time');
 
-// function component() {
-//   const element = document.createElement("div");
+const addCity = async (city) => {
+  const cityKey = await findCity(city);
+  const cityWeather = await findWeather(cityKey.Key);
 
-//   // Lodash, currently included via a script, is required for this line to work
-//   element.innerHTML = _.join(["Hello", "webpack"], " ");
+  return {
+    cityKey,
+    cityWeather,
+  };
+};
 
-//   return element;
-// }
+const display = (info) => {
+  const { cityKey } = info;
+  const { cityWeather } = info;
 
-// document.body.appendChild(component());
+  weatherDetails.innerHTML = `
+                   <h5 class="location">${cityKey.EnglishName}</h5>
+                   <div>${cityWeather.WeatherText}</div>
+                   <div>
+                        <span class="celcius">${cityWeather.Temperature.Metric.Value} &deg;C</span></br>
+                        <span id="fahrenheit">${cityWeather.Temperature.Imperial.Value} &deg;F</span>
+                   </div>`;
 
-const apiKey = "iiT5pz1slpmXaLAWU1cygqoLpwoAeAu3";
+  const imgSrc = cityWeather.IsDayTime ? './asset/day.svg' : './asset/night.svg';
```

Comment:
- [ ] It is a better idea to change the image based on the weather condition i.e. rainy, sunny, cloudy, etc. This is better than either being day or night.